crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -156,6 +156,22 @@ def compute_resources_available?
         true
       end
 
+      # Check for presence of HA setup, which is a requirement for non-disruptive upgrade
+      def ha_presence_check
```

Comment:
As I mentioned here https://github.com/crowbar/crowbar-core/pull/689#discussion_r81117627, the name of this function is confusing and doesn't describe what she is doing. The current name somehow let's assumes the a boolen value is returned.